CONTRIBUTING.md

```diff
@@ -39,3 +40,59 @@ To reactivate the virtual environment:
 (host)$ pip install tox
 (host)$ tox
 ```
+
+
+## Backporting commits
+
+For the purpose of these instructions, it is assumed a remote named `upstream` points to this repository and a remote named `fork` points at your fork. We want to backport commits from `devel` to the `release_2.0` branch.
+
+```
+fork	git@github.com:me/ansible-runner.git (fetch)
+fork	git@github.com:me/ansible-runner.git (push)
+upstream	https://github.com/ansible/ansible-runner.git (fetch)
+upstream	https://github.com/ansible/ansible-runner.git (push)
+```
+
+### Using Cherry Picker 🍒⛏
+
+1. Make sure cherry-picker is installed in the virtual environment.
+
+        pip install cherry-picker
+
+1. Then run `cherry_picker` with the revision that needs backporting.
+
+    **Note:** It is possible to pass multiple branches to `cherry_picker`.
+
+        cherry_picker --pr-remote fork [revision] release_2.0
+
+    **Note:** `cherry_picker` does not currently support backporting merge commits. In this case, use a range of commits before the merge commit.
+
+        cherry_picker --pr-remote fork [start revision]..[end revision] release_2.0
+
+
+### Manually Backporting
+
+1. Create a new branch from the target release branch.
+
+    You may need to first fetch the upstream repository.
+
+        git fetch upstream
+        git checkout --no-track -b my-backport-branch upstream/release_2.0
```

Comment:
Why `--no-track`?

Reply:
Because otherwise, it may track `upstream` while the contributors need to track `origin` usually. If Git tracks some branch on a remote, `git push` w/o the args will push there and those folks will get a "permission denied".

Reply:
Yup, to prevent tracking `upstream/release_2.0`. What you really want is the backport branch to track the backport branch on your fork (once it is pushed there).